pegasus-common/src/main/java/com/linkedin/common/stats/LongTracking.java

```diff
@@ -271,51 +212,4 @@ private void grow()
     _buffer = newBuffer;
     _bufferSize = newBufferSize;
   }
-
-  private static double safeDivide(final double numerator, final double denominator)
-  {
-    return denominator != 0 ? numerator / denominator : 0;
-  }
-
-  public static class Config
```

Comment:
Just want to double check with you that there is no other mps that is using config to create new instance. If not, feel free to directly drop this comment.

Reply:
It only occurs in a test file for LongTracking in the internal `util` mp. Please me know if this class should be added back.

Reply:
That should be fine then, util test should use the util config instead of the pegasus config class.